Remotion/Core/Core/Core.Core.csproj

```diff
@@ -25,17 +25,12 @@
     <Reference Include="System.Configuration" />
   </ItemGroup>
   <!-- Including System.Configuration.ConfigurationManager is required because log4net takes a hard dependency on an older version of this assembly. -->
-  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0-windows'">
-    <PackageReference Include="System.Configuration.ConfigurationManager" Version="6.0.1" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'net7.0-windows'">
-    <PackageReference Include="System.Configuration.ConfigurationManager" Version="7.0.0" />
-  </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'net8.0-windows'">
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="8.0.0-preview.6.23329.7" />
   </ItemGroup>
   <Import Project="..\..\..\Build\Shared.build.targets" />
   <ItemGroup>
+    <ProjectReference Include="..\Development.Analyzers\Core.Development.Analyzers.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" SetTargetFramework="TargetFramework=netstandard2.0" />
```

Comment:
This is new. What has changed that we now need this depedencny?

Bug - it should always have been there?
New Feature? (separate task?)
Sideeffect of the change - What has changed?